pkg/controller/etcdproxy/config.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcdproxy
+
+import (
+	restclient "k8s.io/client-go/rest"
+)
+
+// EtcdProxyConfig type is used to pass information from cli to the controller.
+type EtcdProxyConfig struct {
```

Comment:
is this the EtcdProxyControllerConfig or only for the EtcdProxy (pods)?

Reply:
Fixed. This is for the controller. The struct contains some additional fields, such as kubeconfig, controller namespace name, that are not needed by the EtcdProxy pods, but are needed by the controller.